apps/file_publisher/lib/file_publisher/file_publisher.ex

```diff
@@ -12,11 +12,19 @@ defmodule FilePublisher do
     GenServer.start_link(__MODULE__, file_path, name: __MODULE__)
```

Comment:
Maybe could call `start_link(__MODULE__, file_path)` now? If we even want to keep it.